### DISCUSSION TITLE
Could you provide "typical" machine configurations?

### DISCUSSION THREAD

**Discussion by JKjakay** [86Box/86Box#2605] [General]
86Box does a great job of emulating many different types of PCs from the 80ies and 90ies... But even having personally witnessed much of all this development I find myself in serious trouble every time I try to configure a "typical" Pentium II or a typical 80486 + GUS, etc. I usually have to look it up, and then mistakes occur.

So I wonder, would you be opposed to have a "target machine" option? You already have this concering CPU / Motherboard. What´s keeping you from filling in the other entries too? So if a user selects "PC (1981)" it would set all other configuration options to 64 KB, 360 K Floppies, etc.

Since these "system targets" could be part of the github code set, everybody could join in correcting mistakes, suggesting better configurations, etc. Would really be helpful running these historical systems... 

Could maybe help with bughunting too, since you could always check if it is an edge case or happens with "typical" systems too - just an idea

**Comment by silverlays** [comment]
@JKjakay This feature already exist when creating new machine with the [WinBox for 86Box](https://github.com/86Box/WinBox-for-86Box)

![Create new machine](https://i.imgur.com/SMxxbGl.png)

**Comment by JKjakay** [comment]
Great! Unfortunately, I am on Linux... (Fedora 36). I tried running 86Box in Wine in the past, and it worked, but I had not a lot of success with the frontends, WinBox included. Would be great to have this in some way on Linux too.
